[tests] 24.10 Stage Two tests failing for logs

There's been a few occurances now, and have found that the test below failing, and on restart of the job, it works as normal

```
066-tests_report_tests_plugin_tests_logs.py_JournalSizeLimitTest.test_journal_tailed_and_linked
```

debug.log from avocodo

```
[stdlog] 2024-11-26 08:43:06,719 avocado.test test             L0313 INFO | INIT 1-tests/report_tests/plugin_tests/logs.py:JournalSizeLimitTest.test_journal_tailed_and_linked
[stdlog] 2024-11-26 08:43:06,721 avocado.test parameters       L0141 DEBUG| PARAMS (key=timeout, path=*, default=None) => None
[stdlog] 2024-11-26 08:43:06,721 avocado.test parameters       L0141 DEBUG| PARAMS (key=timeout_factor, path=*, default=1.0) => 1.0
[stdlog] 2024-11-26 08:43:06,721 avocado.test test             L0345 DEBUG| Test metadata:
[stdlog] 2024-11-26 08:43:06,721 avocado.test test             L0347 DEBUG|   filename: /tmp/cirrus-ci-build/tests/report_tests/plugin_tests/logs.py
[stdlog] 2024-11-26 08:43:06,721 avocado.test test             L0353 DEBUG|   teststmpdir: /var/tmp/avocado_30pafaq7
[stdlog] 2024-11-26 08:43:06,721 avocado.test test             L0354 DEBUG|   original timeout: None
[stdlog] 2024-11-26 08:43:06,721 avocado.test test             L0355 DEBUG|   timeout factor: 1.0
[stdlog] 2024-11-26 08:43:06,721 avocado.test test             L0356 DEBUG|   actual timeout: None
[stdlog] 2024-11-26 08:43:06,722 avocado.test test             L0548 INFO | START 1-tests/report_tests/plugin_tests/logs.py:JournalSizeLimitTest.test_journal_tailed_and_linked
[stdlog] 2024-11-26 08:43:06,724 avocado.test stacktrace       L0040 ERROR| 
[stdlog] 2024-11-26 08:43:06,725 avocado.test stacktrace       L0042 ERROR| Reproduced traceback from: /usr/local/lib/python3.12/dist-packages/avocado/core/test.py:638
[stdlog] 2024-11-26 08:43:06,726 avocado.test stacktrace       L0049 ERROR| Traceback (most recent call last):
[stdlog] 2024-11-26 08:43:06,726 avocado.test stacktrace       L0049 ERROR|   File "/tmp/cirrus-ci-build/tests/report_tests/plugin_tests/logs.py", line 85, in test_journal_tailed_and_linked
[stdlog] 2024-11-26 08:43:06,726 avocado.test stacktrace       L0049 ERROR|     self.assertFileExists(tailed)
[stdlog] 2024-11-26 08:43:06,726 avocado.test stacktrace       L0049 ERROR|   File "/tmp/cirrus-ci-build/tests/sos_tests.py", line 355, in assertFileExists
[stdlog] 2024-11-26 08:43:06,726 avocado.test stacktrace       L0049 ERROR|     assert os.path.exists(fname), f"{fname} does not exist"
[stdlog] 2024-11-26 08:43:06,726 avocado.test stacktrace       L0049 ERROR|            ^^^^^^^^^^^^^^^^^^^^^
[stdlog] 2024-11-26 08:43:06,727 avocado.test stacktrace       L0049 ERROR| AssertionError: /var/tmp/avocado_30pafaq7sos_tests.py.JournalSizeLimitTest/sosreport-JournalSizeLimitTest/sosrepor
t-cirrus-task-5545356358844416-2024-11-26-tsuiwhi/sos_strings/logs/journalctl_--no-pager.tailed does not exist
[stdlog] 2024-11-26 08:43:06,727 avocado.test stacktrace       L0050 ERROR| 
[stdlog] 2024-11-26 08:43:06,727 avocado.test test             L0642 DEBUG| Local variables:
[stdlog] 2024-11-26 08:43:06,746 avocado.test test             L0645 DEBUG|  -> self <class 'logs.JournalSizeLimitTest'>: 1-tests/report_tests/plugin_tests/logs.py:JournalSizeLimitTest.test_
journal_tailed_and_linked
[stdlog] 2024-11-26 08:43:06,746 avocado.test test             L0645 DEBUG|  -> tailed <class 'str'>: /var/tmp/avocado_30pafaq7sos_tests.py.JournalSizeLimitTest/sosreport-JournalSizeLimitTes
t/sosreport-cirrus-task-5545356358844416-2024-11-26-tsuiwhi/sos_strings/logs/journalctl_--no-pager.tailed
[stdlog] 2024-11-26 08:43:06,746 avocado.test test             L0740 ERROR| FAIL 1-tests/report_tests/plugin_tests/logs.py:JournalSizeLimitTest.test_journal_tailed_and_linked -> AssertionErr
or: /var/tmp/avocado_30pafaq7sos_tests.py.JournalSizeLimitTest/sosreport-JournalSizeLimitTest/sosreport-cirrus-task-5545356358844416-2024-11-26-tsuiwhi/sos_strings/logs/journalctl_--no-pager
.tailed does not exist
[stdlog] 2024-11-26 08:43:06,746 avocado.test test             L0733 INFO | 
```

The contents of the logs folder

```
total 1336
drwx------ 2 arif arif   4096 Nov 26 08:43 .
drwxr-xr-x 3 arif arif   4096 Nov 26 08:43 ..
-rw-r--r-- 1 arif arif     61 Nov 26 08:43 journalctl_--disk-usage
-rw-r--r-- 1 arif arif 668618 Nov 26 08:43 journalctl_--no-pager
-rw-r--r-- 1 arif arif 602485 Nov 26 08:43 journalctl_--no-pager_--boot
-rw-r--r-- 1 arif arif  66089 Nov 26 08:43 journalctl_--no-pager_--boot_-1
-rw-r--r-- 1 arif arif   4280 Nov 26 08:43 ls_-alZR_.var.log
```

We can see that the file never got created, and hence the test failed. Needs further investigation on what is going on here

--- a/tests/report_tests/plugin_tests/logs.py
+++ b/tests/report_tests/plugin_tests/logs.py
@@ -54,7 +54,7 @@ class JournalSizeLimitTest(StageTwoReportTest):
         from systemd import journal  # pylint: disable=import-error
         _reader = journal.Reader()
         _size = _reader.get_usage() / 1024 / 1024
-        if _size > 30:
+        if _size > 40:
             return
         # write 20MB at a time to side-step rate/size limiting on some distros
         # write over 20MB to ensure we will actually size limit inside sos,

